api-reference/beta/api/profilephoto_get.md

```diff
@@ -4,28 +4,27 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (**profilePhoto** properties).
 
-A GET photo operation first looks for the specified photo in the user's mailbox on Exchange Online, and if 
-it's not available there, then looks in Azure Active Directory (AAD).
+A GET photo operation first attempt to retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory.
```

Comment:
attempts
I also wonder whether you should go further and say - "If the photo is not available in Office 365, or if the user or tenant is not licensed for Office 365,...